akka-testkit-typed/src/test/scala/akka/testkit/typed/scaladsl/BehaviorTestKitSpec.scala

```diff
@@ -76,12 +78,36 @@ object BehaviorTestKitSpec {
 
   }
 
+  object Mutable {
+    class MyMutableBehavior(ctx: ActorContext[String]) extends MutableBehavior[String] {
+      implicit val ec = ctx.system.executionContext
+
+      Future {
+        println("Hoi")
+      }
+
+      override def onMessage(msg: String): Behavior[String] = this
+    }
+
+    def mutable(): Behavior[String] =
+      Behaviors.mutable[String](ctx ⇒ new MyMutableBehavior(ctx))
+  }
 }
 
 class BehaviorTestKitSpec extends WordSpec with Matchers {
+  import BehaviorTestKitSpec._
+  import BehaviorTestKitSpec.Father._
 
   private val props = Props.empty
 
+  "BehaviorTestkit initialization" must {
+    "trigger futures" in {
+      val testkit = BehaviorTestKit[String](Mutable.mutable())
```

Comment:
What's this test for?

Reply:
Seems unrelated, I will remove changes in this file and merge